notebooks_updated/chp_06.md

```diff
@@ -0,0 +1,1803 @@
+---
+jupyter:
+  jupytext:
+    formats: ipynb,md
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.16.0
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+# Code 6: Time Series
+
+```python
+import matplotlib.pyplot as plt
+import arviz as az
+import pandas as pd
+import numpy as np
+
+import tensorflow as tf
+import tensorflow_probability as tfp
+
+tfd = tfp.distributions
+tfb = tfp.bijectors
+root = tfd.JointDistributionCoroutine.Root
+
+import datetime
+print(f"Last Run {datetime.datetime.now()}")
+```
+
+```python
+az.style.use("arviz-grayscale")
+plt.rcParams["figure.dpi"] = 300
+```
+
+## Time Series Analysis as a Regression Problem
```

Comment:
Note: This notebook is failing execution for us at the moment. Use it as a reference. We will have it updated shortly